src/tools/pontoon/ui.md

```diff
@@ -4,115 +4,125 @@
 
 ### Translation workspace
 
-The translation workspace is where strings are translated.
+The translation workspace is where the translation takes place.
+
+![Translation workspace](../../assets/images/pontoon/ui/workspace_standard.png "Screenshot of the standard editor in the translation workspace")
 
-![Translation workspace](../../assets/images/pontoon/ui/workspace_standard.png)
+It’s possible to navigate sequentially through the strings by clicking the `PREVIOUS` or `NEXT` buttons located at the top of the workspace, or by using keyboard shortcuts (`ALT` + arrow down or up). The `COPY LINK` button copies in the clipboard a link to the current string.
 
-A translator can navigate sequentially by clicking the `PREVIOUS` or `NEXT` buttons located at the top of the workspace or by using keyboard shortcuts (`ALT` + arrow down or up).
+Above the editor, the user can view the source string, any comments regarding the string, and the resource path where the string is located.
+In the same area, the `REQUEST CONTEXT or REPORT ISSUE` button can be used to request more information about the current string: it will focus the [COMMENTS section](##source-string-comments), and mention the project manager for the project.
 
-In the upper section of the workspace, the translator can view the source string, any comments regarding the string, and the resource path where the string is located.
+The editor is located in the middle section of the column, and it’s where users can input or edit their translations.
 
-The editing space is located in the lower section of the workspace. The translator can edit the translation or suggestion in the text editor.
+In the lower-right side of the workspace, it’s possible to `COPY` the source string to the editor, `CLEAR` the area where translations are typed, and `SUGGEST` or `SAVE` the translation by clicking the corresponding button. This area is also used to [display warnings and errors](translate.md#quality-checks) when submitting a translation.
 
-To the lower-right side of the workspace, the translator can `COPY` the source string to the editor, `CLEAR` the area where translations are typed, and `SUGGEST` or `SAVE` the translation by clicking the corresponding button. This area is also used to [display warnings and errors](translate.md#quality-checks) when submitting a translation.
+In the lower-left side:
+* Clicking the gear icon allows users to toggle `Translation Toolkit checks` or `Make suggestions`, and navigate to the user settings.
+* Clicking the keyboard icon displays a list of available shortcuts.
+* The numbers displayed to the right of the keyboard icon (e.g. `50|59`) are the number of characters in the source and target string.
 
-To the lower-left side, a translator can access the workspace settings, keyboard shortcuts, and view the number of characters in the target and source string.
+Note that access to some settings is restricted by [user permissions](users.md#user-roles).
```

Comment:
What does that refer to?

Reply:
I thought we were hiding the `Make suggestions` menu for users without translator rights (that's what we're doing in the profile settings).

Right now the behavior is buggy, as the user can toggle `Make suggestions`, but there are no changes.

Should I remove this, or file a bug and have that behavior fixed instead?

Reply:
The note should be moved to the first bullet point then, because that's where we talk about settings.

Yes, please file a bug.